### PR TITLE
Add configuration option to support github output

### PR DESCRIPTION
**Pull request by Nyholm**
I'll make this simple PR so it can quickly get merged which will allow me to be able to run the CI...

It is part of #148

### CODE REVIEW

File: lib/Configuration.php [hunk 1]
```diff
@@ -227,6 +232,16 @@ public function isWarningsAsError(): bool
         return $this->warningsAsError;
     }
 
+    public function getOutputFormat(): string
+    {
+        return $this->outputFormat;
+    }
+
+    public function setOutputFormat(string $outputFormat): void
```
Comment (greg0ire) [inline lib/Configuration.php]:
```suggestion
    /**
     * @psalm-param self::OUTPUT_FORMAT_* $outputFormat
     */
    final public function setOutputFormat(string $outputFormat): void
```

File: lib/Configuration.php [hunk 2]
```diff
@@ -227,6 +232,16 @@ public function isWarningsAsError(): bool
         return $this->warningsAsError;
     }
 
+    public function getOutputFormat(): string
```
Comment (greg0ire) [inline lib/Configuration.php]:
```suggestion
    /**
     * @psalm-return self::OUTPUT_FORMAT_*
     */
    final public function getOutputFormat(): string
```

**Comment by Nyholm** [comment]
Thank you

**Review by greg0ire — APPROVED** [review]

File: lib/Configuration.php [hunk 3]
```diff
@@ -53,6 +55,9 @@ class Configuration
     /** @var bool */
     private $warningsAsError = false;
 
+    /** @var string */
```
Comment (greg0ire) [inline lib/Configuration.php]:
```suggestion
    /** @psalm-var self::OUTPUT_FORMAT_* */
```

File: lib/Configuration.php [hunk 4]
```diff
@@ -28,6 +28,8 @@
 class Configuration
 {
     public const THEME_DEFAULT = 'default';
+    public const OUTPUT_FORMAT_CONSOLE = 'console';
+    public const OUTPUT_FORMAT_GITHUB = 'github';
```
Comment (greg0ire) [inline lib/Configuration.php]:
```suggestion
    public const OUTPUT_FORMAT_GITHUB  = 'github';
```

File: lib/Configuration.php [hunk 5]
```diff
@@ -28,6 +28,8 @@
 class Configuration
 {
     public const THEME_DEFAULT = 'default';
```
Comment (greg0ire) [inline lib/Configuration.php]:
```suggestion
    public const THEME_DEFAULT         = 'default';
```

**Comment by greg0ire** [comment]
Thanks @Nyholm !

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**